doc-site/docs/architecture/zeto.md

```diff
@@ -1,8 +1,122 @@
 # Zeto - Zero Knowledge Proof based tokens
 
-A UTXO based privacy-preserving token toolkit using Zero Knowledge Proofs, implemented via Circom.
+Zeto is a UTXO based privacy-preserving token toolkit for EVM, using Zero Knowledge Proofs, implemented via Circom.
 
 The architecture documentations for Zeto is being managed in a separate Github repository here:
 https://github.com/kaleido-io/zeto
 
-> TODO: Add details of the zeto integration into Paladin (Lead: Jim Zhang)
+Zeto is a growing collection of token implementations that enforce a wide variety of token transaction policies including, but not limited to, mass conservation (for fungible tokens), preservation of asset properties during ownership transfer (for non-fungible tokens), KYC with privacy, and non-repudiation compliance. Each policy is expressed in zeto knowledge proof circuits using [Circom](https://iden3.io/circom). The list of policies and their corresponding token implementations will continue to grow to meet the needs of enterprise use cases.
+
+## Paladin Support
+
+Zeto tokens are natively supported by Paladin, as a domain implementation called "Zeto". The foundational operations of Zeto tokens, `mint`, `transfer` are supported in the initial Paladin release. Support for other operations such as `deposit`, `withdraw` will be added later.
+
+As a client to Zeto tokens, Paladin has the following features built into the single runtime that runs alongside an Ethereum node:
+
+- Tokens indexer: by the nature of a UTXO based design, an account's balance is not known by querying the smart contract, as is the case with ERC-20 tokens. Instead, the UTXOs must be indexed from confirmed onchain transactions in order for an account to know the balance, by adding together all the tokens that belong to that account.
+- Token selector: when sending a transaction that transfers certain amount to another account, the transaction input is made up of a collection of UTXOs that will be spent. The UTXOs must be selected from the unspent UTXOs that the account holds. The selection process needs to take into account the intend transfer amount and the values of the available tokens.
+- ZK proof generator: each Zeto transaction must be accompanied by a ZK proof to demonstrate the validity of the transaction proposal. This is accomplished by a proof generator that is able to use the secrets known only to the Paladin runtime hosting the account key, as private input to the ZKP circuit.
+
+## Private ABI
+
+The private ABI of Zeto is implemented in [Go](https://github.com/LF-Decentralized-Trust-labs/paladin/tree/main/domains/zeto), and can be accessed by calling `ptx_sendTransaction` with `"type": "private"`.
+
```

Comment:
In addition to the methods here, there is also a `lockProof` method on the private Zeto ABI, which should be documented.

Reply:
Unless the "lockProof" on the private ABI is no longer needed, and we can simply use the public Zeto ABI when "lockProof" is needed. Just depends on if any Go logic is needed when locking.